src/routes/api/v1/validators/authValidator.js

```diff
@@ -36,6 +36,18 @@ const email = Joi.object().keys({
     .required(),
 });
 
+const publicAddress = Joi.object().keys({
+  publicAddress: Joi.string()
```

Comment:
We can call it wallet address to be specific

Reply:
Okay

> We can call it wallet address to be specific

Ok done!